extensions/wikia/PhalanxII/PhalanxStatsSpecialController.class.php

```diff
@@ -74,6 +73,23 @@ private function blockStats($blockId) {
 
 		$data['type'] = implode( ', ', Phalanx::getTypeNames( $data['type'] ) );
 
+		/* stats table */
+		$headers = array(
+			wfMsg('phalanx-stats-table-id'),
+			wfMsg('phalanx-stats-table-user'),
+			wfMsg('phalanx-stats-table-type'),
+			wfMsg('phalanx-stats-table-create'),
+			wfMsg('phalanx-stats-table-expire'),
+			wfMsg('phalanx-stats-table-exact'),
+			wfMsg('phalanx-stats-table-regex'),
+			wfMsg('phalanx-stats-table-case'),
+		);
+
+		$tableAttribs = array(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$tableAttribs" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)